src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/xlf/MicrosoftCodeQualityAnalyzersResources.fr.xlf

```diff
@@ -244,7 +244,7 @@
       </trans-unit>
       <trans-unit id="OperatorOverloadsHaveNamedAlternatesCodeFixTitle">
         <source>Add operator overload named alternate</source>
-        <target state="translated">Ajouter une surcharge d'opérateur nommée alternative</target>
+        <target state="translated">Ajouter une surcharge d'opérateur nommée alternate</target>
```

Comment:
@cristianosuzuki77 it looks like the translator might be treating `alternate` as a literal (not translated) term. Can you verify that this term is being translated.